noah_frontend/src/components/donate.js

```diff
@@ -84,7 +89,8 @@ class DonateView extends React.Component {
         let errors = {}
         const phoneRegex = /^\d{10}$/,
               pinCodeRegex = /^\d{6}$/,
-              landPhoneRegex = /^0\d{10}$/
+              landPhoneRegex = /^0\d{10}$/,
+              nameRegex = /^[a-zA-Z][-_ a-zA-Z0-9]+$/
```

Comment:
Looks like same regex is repeated in same file twice , can you write a `isValidName` function

Reply:
Sure. will update it